### ISSUE TITLE
drawings in "find drawing" browser, not always crisp

### ISSUE THREAD

**Issue by totallyRonja** [le-doux/bitsy#110]
The drawings in my drawing browser are slightly blurry since the latest update. Inspecting the element tells me `image-rendering: pixelated` isn't supported. I can't say for sure whether this is a regression, but I didn't notice it previously.

I'm using Firefox Developer Edition 80.0b5 on Windows 10.

![image](https://user-images.githubusercontent.com/10684874/89717659-0daf7080-d9b9-11ea-9ac6-8bfb7b49b323.png)
![image](https://user-images.githubusercontent.com/10684874/89717671-2ddf2f80-d9b9-11ea-899a-f07a35bf77ec.png)

In case thats helping in any way, I usually add this block to my games and that works on most browsers
```css
image-rendering: optimizeSpeed;
image-rendering: -moz-crisp-edges;
image-rendering: -webkit-optimize-contrast;
image-rendering: optimize-contrast;
image-rendering: pixelated;
-ms-interpolation-mode: nearest-neighbor;
```

**Comment by synth-ruiner** [comment]
my pull request #17 will fix this. I can rebase it if needed

**Comment by le-doux** [comment]
sorry for the (very) long wait, but this fix has now been merged into the dev-8-2 branch, so this will get released with v8.2! :D

https://github.com/le-doux/bitsy/commit/f85973599b035efa1edd9b3650d6dda19cce4b99

thanks @synth-ruiner for contributing this - I think you're already on the CREDITS.md page, but let me know if you want to update or change how you're credited (for example if you want to add your github handle)
